packages/theme-check-node/src/NodeFileSystem.ts

```diff
@@ -18,10 +18,7 @@ export const NodeFileSystem: AbstractFileSystem = {
     // console.error('fs/readDirectory', uri);
     const files = await fs.readdir(path.fsPath(uri), { withFileTypes: true });
     return files.map((file) => {
-      return [
-        `${uri}/${file.name}`.replace('file:////', 'file:///'), // fun edge case
-        file.isDirectory() ? FileType.Directory : FileType.File,
-      ];
+      return [path.join(uri, file.name), file.isDirectory() ? FileType.Directory : FileType.File];
```

Comment:
The `/` was causing windows tests to fail, ha! and path.join() also solves the `file:////` problem.